app/src/main/java/com/realityexpander/tasky/agenda_feature/data/common/convertersDTOEntityDomain/EventConverters.kt

```diff
@@ -124,8 +132,7 @@ fun AgendaItem.Event.toEventDTOResponse(): EventDTO.Response {
         isGoing = isGoing ?: false,
         photos = photos
             .map { it.toDTO() }
-            .filterIsInstance<PhotoDTO.Remote>()        // ensure only respond with Remote photos
-        ,
+            .filterIsInstance<PhotoDTO.Remote>(),        // ensure only respond with Remote photos
```

Comment:
In which case would a `PhotoDTO` contain local photos?

Reply:
It should never contain them, this is just a sanity check, and also conveying intent about what is going on. 

Is this a bad practice?

Reply:
No it of course makes sense to make sure only local photos can be uploaded, but I'm wondering why you don't just model your `PhotoDTO` like this to avoid that local photos can be passed at all:
```
data class PhotoDTO(
    val photos: List<PhotoDTO.Remote>
)
```

Reply:
Yes showing intent is great, I just don't know if I'd do it by implementing an unused class here (`PhotoDTO.Local`). What do you think about something like this?
```
data class RemotePhotoDTO(
    val url: String, 
    val key: String
)

data class EventDTO(
    val photos: List<RemotePhotoDTO>
    ...
)
```

Reply:
OK, maybe I was mistaken about what the PhotoDTO is used for. 

`PhotoDTO.Local` contains the URIs that are decoded and sent to the server. 

When a api response come back, the `PhotoDTO.Remote` is populated with links to server images. You cannot convert a `PhotoDTO.Local` to a domain or entity object.

Reply:
Yeah you can do it like that :)
I was just curious, since your data layer could also perfectly access a model from domain (like `Photo.Local` here), but I see the point of following consistency here.

Reply:
OK cool!